package.json

```diff
@@ -16,26 +16,27 @@
     "LICENSE"
   ],
   "scripts": {
-    "test": "jshint lib index.js && jscs lib index.js && mocha -t 5000 -b -R spec test/index",
-    "cover": "nyc --reporter=lcov --reporter=text-summary npm test"
+    "lint": "eslint .",
+    "test": "jshint lib index.js && mocha -t 5000 -b -R spec test/index",
```

Comment:
remove jshint, create a `pretest` script that calls the linter (like https://github.com/gulpjs/sparkles/blob/master/package.json#L21)